Resources/Documentation/installation_and_setup.md

```diff
@@ -34,7 +29,18 @@ brew upgrade
 ```
 (This command will upgrade all of the programs you've installed through Homebrew.)
 
-Thus far installing vrecord on Linux using Linuxbrew has not been successful.
+### Linux
+
+As of version 2019-01-19, vrecord can now be installed on select Linux distributions, most easily via [Linuxbrew](https://docs.brew.sh/Homebrew-on-Linux), with the same macOS Homebrew Terminal commands:
+```
+brew tap amiaopensource/amiaos
+brew install vrecord
+```
+But if find yourself experiencing Linux installation issues, it may be beneficial to first install vrecord's core dependencies, and then install vrecord without dependencies:
```

Comment:
maybe add a caveat about how this method will only support core functions (things like qcli will still need to be installed before they work)